composer.json

```diff
@@ -17,7 +17,8 @@
     "fond-of-spryker/brand": "dev-master",
     "spryker/data-import": "^1.3.0",
     "spryker/kernel": "^3.0.0",
-    "spryker/propel-orm": "^1.0.0"
+    "spryker/propel-orm": "^1.0.0",
+    "fond-of-codeception/spryker": "^1.0"
```

Comment:
"fond-of-codeception/spryker": "^1.0" in require? require-dev ist the correct place.

Reply:
Thanx for the notice. I've corrected.